### PR TITLE
fixed  #2174 Issue

### LINKED ISSUE #2174: Give page the right lang attribute
### Is your feature request related to a problem? Please describe.

Currently, the page always have `<html lang="en">` regardless of what language the UI is using.

### Describe the solution you'd like

Each language in [resources/i18n](https://github.com/navidrome/navidrome/tree/2400e4f60da705888298cb493948b1c008a1980c/resources/i18n) should include an [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag), for example, Japanese is `ja`, Simplified Chinese is generally `zh-CN` and Traditional Chinese is `zh-TW`.

When the UI language is updated, the `lang` attribute should be updated to the corresponding language tag.

### Describe alternative solutions that would also satisfy this problem

Not aware of.

### Additional context

This is a problem because language attribute affects
* a11y, e.g. screen reader,
* fallback font choice,
  * which is especially a problem for CJK languages as they share Chinese characters, but different fonts have different character coverage, and the same character can come with different shape in different languages,
* text transform, e.g. uppercasing,
  * which is especially a problem for Latin-based languages, because different languages have different transformation rules, for example,
    * in Turkish, the uppercase of "i" is "İ" rather than "I"
    * in Irish, the uppercase of "gc" is "gC" not "GC"

(Though my main concern is on CJK.)

### PR DESCRIPTION
**Pull request by subhajit20**
fixed  #2174 Issue

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -142,6 +142,8 @@ const Admin = (props) => {
 }
 
 const AppWithHotkeys = () => {
+  let language = localStorage.getItem('locale');
```
Comment (deluan) [inline ui/src/App.js:145]: We need to have a default here:
```
      const language = localStorage.getItem('locale') || 'en';
```

**Review by deluan — CHANGES_REQUESTED** [review]
LGTM. Just a small change.

File: ui/src/App.js (hunk 1 above)
Comment (subhajit20) [reply ui/src/App.js:145]: Okay sure wait a moment

Comment (subhajit20) [reply ui/src/App.js]: Yeah, have changed that and committed.
please check once.

**Comment by subhajit20** [comment]
@deluan  hello is their problem to merge my branch?

**Comment by deluan** [comment]
> @deluan hello is their problem to merge my branch?

Yes, the pipeline is broken. Check the error, seems to be something with prettier.

**Comment by subhajit20** [comment]
@deluan will you fix it ?

**Comment by deluan** [comment]
No, it is your code ;) Click the `details`  in the failed check above and read what is wrong.

Screenshot 2023-04-06 at 11 06 47 AM

**Comment by subhajit20** [comment]
> No, it is your code ;) Click the `details` in the failed check below and read what is wrong.
> 
> Screenshot 2023-04-06 at 11 06 47 AM

@deluan hello I have committed once again can you check that once again.

**Comment by deluan** [comment]
Hey, you just need to run prettier in your files, not the whole project! Can you please revert the last commit and just run:
```
cd ui
npm run prettier
```
This will run prettier against the UI code only. Thanks.

**Comment by deluan** [comment]
To avoid this kind of issues in the future, please setup your environment following the instructions from the docs: https://www.navidrome.org/docs/developers/dev-environment/

Seems that you missed running `make setup` (step 6 of the page above), which installs git hooks that run theses checks in your machine before pushing to GitHub.

**Comment by deluan** [comment]
Sorry, when you ran prettier on all files of the project, you broke a lot of stuff. I suggest you discard this PR and create a new one, with just your changes. Don't forget to run `make setup` in your local environment (and check the docs on how to setup a local environment to be sure you are not missing anything).

**Comment by subhajit20** [comment]
> Sorry, when you ran prettier on all files of the project, you broke a lot of stuff. I suggest you discard this PR and create a new one, with just your changes. Don't forget to run `make setup` in your local environment (and check the docs on how to setup a local environment to be sure you are not missing anything).

okay sure

**Pull request closed without merging**